src/server/models/User.js

```diff
@@ -9,13 +9,28 @@ export default class User extends BaseModel {
       ...BaseModel.jsonSchema.required,
       'githubId',
       'name',
-      'email',
```

Comment:
Yes it's empty if the sync creates the user, registered users can be identified with accessToken